Controllers/WordsController.cs

```diff
@@ -63,5 +137,31 @@ public async Task<IActionResult> PostWord(Word word)
 
             return Ok(word);
         }
+
+        public async Task<List<Word>> AllWords()
+        {
+            var allWords = await _context.Words.ToListAsync();
+            return allWords;
+        }
+
+        public async Task<Word> GetWordById(Guid id)
+        {
+            var word = await _context.Words.FindAsync(id);
+
+            if (word == null)
+            {
+                throw new NullReferenceException();
+            }
+
+            return word;
+        }
+
+        public async Task<String> GetWotd()
```

Comment:
Better not to use custom abbreviations. They can not be obvious to other developers